File: SCALA/Cap07/c07ex07.scala
// c07ex07.scala

import scala.io.StdIn._

object Main {
  def igual(A: Int, B: Int): Boolean = {
    A == B
  }

  def main(args: Array[String]): Unit = {
    var x, y: Int = 0	  
	  
    print("Informe o 1o. valor: ")
    x = readInt()

    print("Informe o 2o. valor: ")
    y = readInt()

    if (igual(x, y)) {
      println("Valores sao iguais")
    } else {
      println("Valores sao diferentes")
    }

    print("\nTecle <Enter> para encerrar... ")
    readLine()
  }
}

File: DART/Cap07/c07ex07.dart
// c07ex07.dart

import 'dart:io';

bool igual(int A, int B) {
  return A == B;
}

void main() {
  int x;
  int y;

  stdout.write("Informe o 1o. valor: ");
  x = int.tryParse(stdin.readLineSync() ?? '') ?? 0;

  stdout.write("Informe o 2o. valor: ");
  y = int.tryParse(stdin.readLineSync() ?? '') ?? 0;

  if (igual(x, y)) {
    print("Valores sao iguais");
  } else {
    print("Valores sao diferentes");
  }

  stdout.write("\nTecle <Enter> para encerrar... ");
  stdin.readLineSync();
}
